scss/titles.scss

```diff
@@ -0,0 +1,23 @@
+@import 'color';
+
+.title {
+  color: $color-black;
+  font-size: 24px; 
+  position: relative;
+  left: 90px;
+  top: 60px;
```

Comment:
This is a relative position element, dont use left or top in this one. To move it use margin or padding. But dont focus on this in this PR, just remove them (top and left). We will see how to position the title once we start adding things